### PR TITLE
Update esql-getting-started.ipynb

### PR DESCRIPTION
**Pull request by leemthompo**
Closes https://github.com/elastic/elasticsearch-labs/issues/207

### CODE REVIEW

File: notebooks/esql/esql-getting-started.ipynb [hunk 1]
```diff
@@ -1156,7 +1156,7 @@
     "client.enrich.put_policy(name=\"clientip_policy\", body=policy)\n",
     "\n",
     "# Execute the enrich policy without waiting for completion\n",
```
Comment (miguelgrinberg) [inline notebooks/esql/esql-getting-started.ipynb:1158]: Should the comment be updated as well?

Comment (leemthompo) [reply notebooks/esql/esql-getting-started.ipynb:1158]: It certainly should, thanks

Comment (leemthompo) [inline notebooks/esql/esql-getting-started.ipynb:1158]:
```suggestion
    "# Execute the enrich policy and wait for completion\n",
```

**Review by miguelgrinberg — APPROVED** [review]
LGTM!

**Pull request merged**